setup.py

```diff
@@ -27,7 +27,8 @@ def unique_flatten_dict(d):
     'docs': ['sphinx==3.4.3', 'docutils==0.16', 'sphinx_autodoc_typehints==1.11.1', 'sphinx-rtd-theme==0.5.1', 'Jinja2<3.1'],
     'test': ['flake8>=5.0', 'mock', 'mypy', 'pytest'] + stubs,
     'testai': [
-      'numba>=0.57.1'  # https://github.com/numba/numba/issues/8615
+      'numba>=0.57.1',  # https://github.com/numba/numba/issues/8615
+      'scikit-learn<=1.3.2'
```

Comment:
Also > the known bad version?

Reply:
scikit-learn<=1.3.2,>1.4.0 give:
ERROR: No matching distribution found for scikit-learn<=1.3.2,>1.4.0
I use not equal instead